var/spack/repos/builtin/packages/py-torch/package.py

```diff
@@ -114,6 +114,11 @@ class PyTorch(PythonPackage, CudaPackage, ROCmPackage):
     # https://github.com/pytorch/pytorch/issues/80805
     conflicts("+openmp", when="platform=darwin target=aarch64:")
 
+    # https://github.com/pytorch/pytorch/issues/97397
+    conflicts(
+        "~tensorpipe", when="+distributed", msg="TensorPipe must be enabled with +distributed"
```

Comment:
```suggestion
        "~tensorpipe", when="@1.8: +distributed", msg="TensorPipe must be enabled with +distributed"
```
According to https://github.com/pytorch/pytorch/issues/97397